ovl_tool_gui.py

```diff
@@ -17,16 +17,24 @@ class MainWindow(widgets.MainWindow):
 	def __init__(self):
 		widgets.MainWindow.__init__(self, "OVL Tool", )
 		
-		self.ovl_data = OvlFormat.Data()
+		self.ovl_data = OvlFormat.Data(progress_callback = self.update_progress)
 
 		supported_types = ("DDS", "PNG", "MDL2", "TXT", "FGM", "FDB", "MATCOL", "XMLCONFIG", "ASSETPKG", "LUA")
 		self.filter = "Supported files ({})".format( " ".join("*."+t for t in supported_types) )
 
 		self.file_widget = widgets.FileWidget(self, self.cfg)
 		self.file_widget.setToolTip("The name of the OVL file that is currently open.")
 
-		self.e_name_pairs = [ (QtWidgets.QLineEdit("old"), QtWidgets.QLineEdit("new")) for i in range(3) ]
+		#self.e_name_pairs = [ (QtWidgets.QLineEdit("old"), QtWidgets.QLineEdit("new")) for i in range(3) ]

```

Comment:
Got rid of this because it didn't actually seem to do anything other than take up GUI space.

Reply:
This is mainly for dev purposes right now, we haven't actually put it to proper use yet. No matter what we try, we can't seem to add new files to an archive...